src/WebJobs.Script/ScriptConstants.cs

```diff
@@ -179,6 +179,8 @@ public static class ScriptConstants
         public const string Windows64BitRID = "win-x64";
         public const string Windows32BitRID = "win-x86";
         public const string WorkFlowAppKind = "workflowapp";
+        public const string LogicAppDefaultExtensionBundleVersion = "[1.*, 2.0.0)";
```

Comment:
It's not good practice to couple Functions host with Logic Apps this way.  Prefer if the user (Logic Apps infra) configured the bundle explicitly.

Reply:
discussed offline.  Could you please add comments around this explaining that this is temporary until we support configuring host.json at app creation time (via ARM template / REST API at minimum)